sim-lib/src/lib.rs

```diff
@@ -25,10 +25,9 @@ mod random_activity;
 mod serializers;
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(untagged)]
```

Comment:
TIL of this magic! 🥇

Reply:
Oh yeah, there's a bunch of these you can use depending on what you're aiming for:

https://serde.rs/enum-representations.html
https://serde.rs/attr-flatten.html